applications/bountied_whistleblowing.md

```diff
@@ -0,0 +1,103 @@
+
+- Technology Stack:
+  - Frontend: Next.js
+  - Backend: Node.js with Express
+  - Authentication: Auth0/Firebase
+  - Database: PostgreSQL/MySQL, MongoDB
+  - Caching: Redis
+  - Search Engine: Elasticsearch/Algolia
+  - Blockchain: Ethereum (for smart contracts)
+  - ZK Proofs: Circom (ported from Circom)
+  - Cryptography: RSA, SHA
+  - Privacy Layer: polygon Network
+
+
+## [Section 3] Ecosystem Fit
+- Similar Projects:
+While there are existing whistleblowing platforms, this project uniquely combines zero-knowledge proofs, blockchain technology, and financial incentives to create a secure and efficient marketplace for sensitive information.
+
+- Unique Contribution:
+This project brings enhanced privacy and security to whistleblowing through zero-knowledge proofs and blockchain technology. It addresses critical issues in existing platforms while creating financial incentives for high-quality information, potentially setting a new standard for secure information exchange in journalism and corporate compliance.
+
+## [Section 4] Team :busts_in_silhouette:
+    Team Members: Yash Bharti, Krishanu Dhar
+
+- Contact Information:
+  - Name: Yash Bharti
+  - Email: yashbharti924@gmail.com
+  - Name: Krishanu Dhar
+  - Email: rony.kris@gmail.com
+
+
+- Prior Work/Research Yash:
+  - Experience with Noir Circuits (PSE Grant)
+  - Porting Circuits from Circom to Noir
+  - Smart Contracts with Solidity
+  - ZK Circuit Writing: Halo2, Circom
+  - NYU BA CS '22, University of Cambridge - MPhil (CBL Lab) - Deferred 2025
+  - AI Research Background - Deep Reinforcement Learning for Robotics, Natural Language Understanding, Cognitive Science
+- Prior Work/Research Krish:
+   -  Full stack, smart contract and zkp developer
+   -  6+ years in web3
+   -  Solidity developer on EVM chains
+   -  ZK circuits developer in Mina, and polygon
+   -  Built zkpoker on EVM with solidity and zokrates
+   -  zknonogram and zkuno using Noir on polygon
+   -  GitHub: https://github.com/ronykris
+
+## [Section 5] Development Roadmap :open_book:
+
+### Milestone 1 — Planning and Initial Development
+- Estimated Duration: 1 month
+- Description: 
+  - Project planning and environment setup
+  - Port Circom circuits to Circom
+  - Develop ZK Email Verifier with RSA/SHA integration
+  - Implement client-side proving functions in Circom
+  - Develop regex mapping for search terms
+- FTE: 1 FTE
+- Costs: 7,500 USDC
+  - Development labor: 6,000 USDC
+  - Software licenses and tools: 1,000 USDC
+  - Miscellaneous expenses: 500 USDC
+
+### Milestone 2 — Integration and Deployment
+- Estimated Duration: 1 month
+- Description:
+  - Generate and integrate ERC-721 tokens
+  - Complete integration with polygon Network
+  - Develop financial transaction features
+  - Comprehensive testing of all components
+  - Deploy ZK Email Verifier and related components
+  - Finalize documentation and user guides
+- FTE: 1 FTE
+- Costs: 7,500 USDC
```

Comment:
I dont see why you need another month for this.

1. Generating an ERC721 is a fast task
2. Integrating with Polygon and Deploying zkEmail verifier should be fast as well once you have the tests

Mostly what it seems is that you will do testing and documentation, for this this extra month seems a bit much